src/main/java/org/folio/circulation/domain/Loan.java

```diff
@@ -226,11 +241,13 @@ public String getActionComment() {
 
   public Result<Void> isValidStatus() {
     if (!representation.containsKey(STATUS)) {
+      log.warn("isValidStatus:: Loan does not have a status");
```

Comment:
Can we extract message into a variable and reuse it?

Reply:
done